stories/MewTokenContainer/MewTokenContainer.stories.js

```diff
@@ -35,6 +35,9 @@ export const MEWTokenContainer = () => ({
     size: {
       default: optionsKnob('size', sizeOptions, sizeOptions.small, { display: 'inline-radio' })
     },
+    borderSize: {
+      default: boolean('border-size', false)
```

Comment:
shouldn't this be text with default value?

Reply:
Fixed